packages/manager/modules/pci/src/projects/project/private-registry/credentials/credentials.controller.js

```diff
@@ -0,0 +1,56 @@
+import get from 'lodash/get';
+
+export default class PrivateRegistryDeleteCtrl {
+  /* @ngInject */
+  constructor(
+    $stateParams,
+    $translate,
+    $window,
+    pciPrivateRegistryService,
+  ) {
+    this.registryId = $stateParams.registryId;
```

Comment:
Instead of retrieving `registryId`, you should retrieve registry object (resolve from state `/registry/{id}` for instance)

In fact, the main objective is to avoid doing a lot a same call to API:

`/registry/{id}` -> do a call on API to get registry
`/registry/{id}/myAction` -> do a call on API to get registry
`/registry/{id}/myAction2` -> do a call on API to get registry

If you resolve your registry in first state:

`/registry/{id}` -> do a call on API to get registry
`/registry/{id}/myAction` -> No call needed, we already have `registry` in resolve
`/registry/{id}/myAction2` -> No call needed, we already have `registry` in resolve

I know this pattern is not always applied in public-cloud, but we should use it * :)